colors/highlite.lua

```diff
@@ -857,12 +857,12 @@ local highlight_groups = {
 	NERDTreeLinkTarget = 'Tag',
 
 	--[[ 4.4.8. nvim-treesitter ]]
-	TSConstBuiltin = 'TSConstant',
-	TSConstructor = 'TSFunction',
-	TSDanger = 'ErrorMsg',
-	TSFuncBuiltin = 'TSFunction',
-	TSTag = 'Tag',
-	TSWarning = 'WarningMsg',
+	['@constant.buildin'] = '@constant',
+	['@constructor'] = '@function',
+	['@text.danger'] = 'ErrorMsg',
+	['@function.buildin'] = '@function',
```

Comment:
Is this a typo (i.e. should it be `@function.builtin`)?

Reply:
Oh, yes. It seems that I have typed the wrong word.

Reply:
I will fix it and recommit it soon.